### ISSUE TITLE
what is the top-1 acc for weights/hardnet_petite_base.pth on imagenet

### ISSUE THREAD

**Issue by Libaishun** [PingoLH/FCHarDNet#32]
what is the top-1 acc for weights/hardnet_petite_base.pth on imagenet

**Comment by PingoLH** [comment]
We actually didn't fully train the petite model with ImageNet. The training was early stopped at around 100th epoch out of 150 epochs, and unfortunately, we didn't record the accuracy.

**Comment by LaCandela** [comment]
What a pitty, I am also interested in the top-1/top-5 accuracy of this pre-train...
